lib/hawkular/inventory/entities_v4.rb

```diff
@@ -13,6 +13,14 @@ def initialize(hash)
       @type = hash['type']
       @properties = hash['properties'] || {}
     end
+
+    def hawkular_id
```

Comment:
Should `hawkular_id` and `hawkular_type` be always present? If so it's better to use `fetch` instead of `[]`.

Reply:
They should be always present... for now. Not later when prometheus comes in the game.

Reply:
Is prometheus related to inventory as well? Thought it was supposed to be contained in Metrics only. Either way I don't think it is a good idea to return nil on this.

Reply:
This is a "step" that i suppose we are going to remove when prometheus migration is done.
Each resource has a list of metrics, and this tells the "metric_id" where this metric is stored.

e.g. Resource A and B have both a x_count, but A.x_count is stored in `hawk_{a.id}_{x.id}.{hash}` and b in `hawk_{a.id}_{x.id}.{hash}`.